code/modules/vehicles/multitile/cm_armored.dm

```diff
@@ -341,7 +343,7 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 /atom/proc/tank_collision(obj/vehicle/multitile/hitbox/cm_armored/C, facing, turf/T, turf/temp)
 	return
 
-/mob/living/tank_collision(obj/vehicle/multitile/hitbox/cm_armored/C, facing, turf/T, turf/temp)
+/mob/living/tank_collision(obj/vehicle/C, facing, turf/T, turf/temp)
```

Comment:
```suggestion
/mob/living/tank_collision(obj/vehicle/V, facing, turf/T, turf/temp)
```
Try using variables that represent what you are after. It improves readability.
`C` usually implies `carbon`. Or the more evil `cm_armored` we are trying to get rid of.